kvmagent/kvmagent/test/shareblock_testsuite/test_sblk_metadata.py

```
@@ -0,0 +1,1464 @@
+        finally:
+            self._cleanup_lv(meta_path)
+
+    # == prefix_rebase_backing_files ==========================================
+
+    def _test_rebase_empty_file_paths_returns_zero(self):
+        count = sblk_prefix_rebase_backing_files(
+            [], '/dev/old_vg', '/dev/new_vg', translate_absolute_path_from_install_path, lvm)
+        self.assertEqual(count, 0)
+
+    def _test_rebase_empty_old_prefix_raises(self):
+        with self.assertRaises(Exception):
```

Ruff (0.15.7)

[warning] Do not assert blind exception: `Exception`

(B017)